### DISCUSSION TITLE
Event when soft keyboard is shown or hidden

### DISCUSSION THREAD

**Discussion by FM1973** [dotnet/maui#21682] [Q&A]
Can anyone guide me in the right direction to detect when the soft keyboard is being shown or hidden. I´d like to subscribe an event on Android and IOS. The optimum would be to get the event with the height of the soft keyboard.

Thanks!

**Comment by awasilik** [comment]
No actions since April 6th :( 
That's something that would really be beneficial :)

**Reply by ghost** [reply]
For us, we subscribed to clicking on elements outside of this entry and then asking:

```
if (Entry.IsSoftInputShowing())
{
     Entry.HideSoftInputAsync();
}
```

I know it's a workaround, but for a start.
